jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx

```diff
@@ -245,6 +272,9 @@ class QueryTextEditor extends React.Component<
   }
 
   handleEditorDidMount(_, editor) {
+    if (this.editorRef.current) {
+      this.setState({ height: this.editorRef.current.clientHeight });
+    }
```

Comment:
The `editor.layou({width, height})` call needs to know a height along with a width, so here is where we read/set the current height in order to pass that into the resize dimensions.